crates/sui-benchmark/tests/simtest.rs

```diff
@@ -554,7 +554,12 @@ mod test {
             config
         });
 
-        let test_cluster = build_test_cluster(4, 5000).await;
+        let test_cluster = Arc::new(
+            init_test_cluster_builder(4, 5000)
+                .with_num_unpruned_validators(2)
+                .build()
+                .await,
```

Comment:
maybe just add this option to build_test_cluster? i think it can affect any test

Reply:
Yeah I was a bit hesitant because it _can affect_ any test. For example, I am not sure if in some cases we would like to exercise the fact that there is no full history. Can't think of such test, but if you believe I am overthinking then happy to make the change.

Reply:
I've ended up exposing as an option since some tests need to test the pruning path.